crowdgezwitscher/crowdgezwitscher/urls.py

```diff
@@ -40,7 +40,7 @@
 
     # User area URLs
     url(r'^$', views.landingpage, name='landingpage'),
-    url(r'^map/$', views.index, name='index'),
-    url(r'^map/(?:contact)?/?$', views.index, name='index'),
-    url(r'^imprint/$', views.index, name='index'),
+    url(r'^map/$', views.index, name='map'),
```

Comment:
Meinst du in der Zeile  `url(r'^imprint/$', views.index, name='imprint'),` ?
Das andere kam durchs reinmergen des Develops, da hab ich nichts dran geändert, kann es aber gerne auch noch mit anpassen